compiler/tools/suggest.nim

```diff
@@ -484,6 +484,9 @@ proc findTrackedSym*(g: ModuleGraph;): PSym =
 
 proc executeCmd*(cmd: IdeCmd, file, dirtyfile: AbsoluteFile, line, col: int;
              graph: ModuleGraph) =
+  ## executes the given suggest command for a given file,
+  ## at the position described by line and column.
+  ## If dirtyFile is non-empty, then its contents are used as part of the analysis.
```

Comment:
```suggestion
  ## executes the given suggest command, `cmd`, for a given `file`, at the
  ## position described by `line` and `col`umn. If `dirtyFile` is non-empty,
  ## then its contents are used as part of the analysis.
```

comments and lines in general should be formatted to 80 chars, and backticks should be used to identify/highlight parameter names.